app/src/main/java/org/mozilla/fenix/perf/Stat.kt

```diff
@@ -25,6 +26,7 @@ private const val FIELD_POS_STARTTIME = 21 // starttime naming matches field in
 open class Stat {
 
     @VisibleForTesting(otherwise = PRIVATE)
+    @Throws(IOException::class)
```

Comment:
nit: `@Throws` appears to be designed for Java interop only which doesn't really apply to us ([source](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-throws/)):

> This annotation indicates what exceptions should be declared by a function when compiled to a JVM method.

I'd change this to a kdoc comment instead, i.e.:
```kotlin
/**
 * @throws [java.io.FileNotFoundException]
 */
```

I'd also use the most specific exception possible. We should also document this on the public facing method, `getProcessStartTimeTicks`

Reply:
I was looking in the code that  explicitly throws this exception, and I thought maybe another privacy manager would choose to throw another `IOException`. I will use the most specific we have for now, but if in the future it shows another, we should change it to more general _IOException_.